SpaceCommander.podspec

```diff
@@ -14,5 +14,8 @@ Pod::Spec.new do |s|
   # Use this file to overcome CocoaPods limitation that at least one source file must be specified.
   s.source_files = "CocoaPodsDummy.m"
 
-  s.preserve_paths = "**/*"
+  s.preserve_paths = "**/*", ".clang-format"
+  
+  # Run setup-repo.sh when the pod is installed.
+  s.prepare_command = "./setup-repo.sh"
```

Comment:
It runs straight from the directory the pod was installed so its relative to the path!